index.Rmd

```diff
@@ -67,24 +66,20 @@ The following is intended as a set of tips for users of the OpenSAFELY system at
 
 - As per the study definition best practice, outputs from the cohortextractor should be provided as `.rds` files
 
-- Use the **readr** package to read `.rds` files 
+- Use the **readr** package to read and write `.rds` files 
 
     ``` r
-    library(readr)
-    read_rds(file = "mydata.rds")
+    object <- readr::read_rds(file.path("output", "mydata.rds"))
+    readr::write_rds(object, file.path("output", "mydata.rds"), compress = "gz")
     ```  
     
 <!--
   - Alternatively, use the **vroom** package, the `vroom::vroom()` function has the `col_sel` argument
 -->
-
-- Use base R to save `.rds` files
-
-    ``` r
-    saveRDS(object, file = "mydata.rds")
-    ```  
     
-- All `.rds` files should all be classified as `highly_sensitive` in actions
+- All `.rds` files should all be classified as `highly_sensitive` in actions, as they cannot be opened on the results server and therefore cannot be reviewed or released.
+- `readr::write_rds` does not compress files by default. However, for large files it is recommended to compress using the argument `compress = "gz"`.
```

Comment:
Any thought on what constitutes 'large' files? Might be good to have a rule of thumb to follow.

Reply:
I think suggest to use the compression always as then we don't have to worry about a rule of thumb (which would probably be something like 10 MB or 100 MB).

Reply:
Yes true. In that case, really no reason to to recommend `readr`, other than for me enjoying the tidyverse syntax!

Reply:
I'm happy with recommending **readr**. I did have a look at the **readr** rds functions [here](https://github.com/tidyverse/readr/blob/a14f189a344f60113540d75f13b49432df12bb70/R/rds.R). They are just simple wrappers using the base R functions (so no speed up).

Reply:
Yes, no speed up for write_rds, but readr::write_csv faster than utils::write.csv. 

I've updated the comment about compressing .rds files to avoid ambiguity.

Let me know if you're happy with the other changes and I'll merge.